### ISSUE TITLE
Your model loss function

### ISSUE THREAD

**Issue by am-h** [rikdz/GraphWriter#15]
Hi 
Would you please share the model loss function ? you mention it is negative joint log likelihood. but I need it in details.

Many thanks

**Comment by wutaiqiang** [comment]
l = F.nll_loss(**) Line 73 in train.py
